spec/imsc-hrm.html

```diff
@@ -280,9 +280,7 @@ <h2>General</h2>
       document</a> E<sub>n</sub> using the following steps in order:</p>
 
       <ol>
-        <li>clear the pixels, except for the first <a>intermediate synchronic document</a> E<sub>0</sub> for the which the pixels
-        of P<sub>0</sub> SHALL be assumed to have been cleared;
-        </li>
+        <li>clear the pixels of the entire <a>Root Container Region</a>, unless n=0 or E<sub>n-1</sub> was empty;</li>
```

Comment:
Do we need to make any provision for consecutive document playback, e.g. what happens in a segmented delivery scenario? If we do, possibly `n=0` would need to be adjusted, or we would need some ISD comparison and deduplication. Possibly a separate issue.

Reply:
I don't think it's entirely clear what "empty" means here.

Reply:
> I don't think it's entirely clear what "empty" means here.

Will add a definition.

Reply:
>  or we would need some ISD comparison and deduplication. Possibly a separate issue.

Please file an issue

Reply:
@nigelmegitt Definition for _empty ISD_ added at https://github.com/w3c/imsc-hrm/pull/9/commits/b17fe84f4bb21cc5a8da0bb2cef1a499f509abaa